app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,17 @@
+<nav class="osra-nav-header" role="navigation">
+  <div class="osra-header-inner">
+      <div class="osra-header-primary">
+        <p>OSRA Support System</p>
+      </div>
+      <div class="osra-header-secondary">
+        <ul class="osra-header __secondary">
+          <% if admin_user_signed_in?  %>
```

Comment:
I can do that, however, the difficult part is to test when the user is already logged in, that should be the next part.